docs/server/source/production-deployment-template/workflow.rst

```diff
@@ -74,112 +76,32 @@ We'll say more about that file below.)
-    ],
-    "MongoDB_Monitoring_Agent": [
-      "mdb-mon-instance-1",
-      "mdb-mon-instance-2",
-      "mdb-mon-instance-3",
-      "mdb-mon-instance-4"
-    ],
-    "Tendermint": [
-      "tm-instance-1",
-      "tm-instance-2",
-      "tm-instance-3",
-      "tm-instance-4"
-    ]
-  }
-
-
-☐ Generate three keys and corresponding certificate signing requests (CSRs):
-
-#. Server Certificate for the MongoDB instance
-#. Client Certificate for BigchainDB Server to identify itself to MongoDB
-#. Client Certificate for MongoDB Monitoring Agent to identify itself to MongoDB
-
-Use the self-signed CA to sign those three CSRs. For help, see the pages:
-
-* :doc:`How to Generate a Server Certificate for MongoDB <../production-deployment-template/server-tls-certificate>`
-* :doc:`How to Generate a Client Certificate for MongoDB <../production-deployment-template/client-tls-certificate>`
-
-☐ Make up an FQDN for your BigchainDB node (e.g. ``mynode.mycorp.com``).
+2. Make up an FQDN for your BigchainDB node (e.g. ``mynode.mycorp.com``).
 Make sure you've registered the associated domain name (e.g. ``mycorp.com``),
 and have an SSL certificate for the FQDN.
 (You can get an SSL certificate from any SSL certificate provider.)
 
-☐ Ask the BigchainDB Node operator/owner for the username to use for authenticating to
-MongoDB.
+3. Download the HTTPS certificate chain and HTTPS certificate key of your registered domain.
+Certificate chain includes your primary SSL cert (e.g. your_domain.crt) followed by all intermediate and root
+CA cert(s). e.g. If cert if from DigiCert, download "Best format for nginx".
+
+4a. If the BigchainDB node uses 3scale for API authentication, monitoring and billing,
+you must ask the BigchainDB node operator/owner for all relevant 3scale credentials and deployment
+workflow.
 
-☐ If the cluster uses 3scale for API authentication, monitoring and billing,
-you must ask the BigchainDB node operator/owner for all relevant 3scale credentials -
-secret token, service ID, version header and API service token.
+4b. If the BigchainDB does not use 3scale for API authentication, then the organization managing the BigchainDB
+node **must** generate a unique *SECRET_TOKEN* for authentication and authorization of requests to the BigchainDB node. 
 
-☐ If the cluster uses MongoDB Cloud Manager for monitoring,
-you must ask the managing organization for the ``Project ID`` and the
-``Agent API Key``.
-(Each Cloud Manager "Project" has its own ``Project ID``. A ``Project ID`` can
-contain a number of ``Agent API Key`` s. It can be found under
-**Settings**. It was recently added to the Cloud Manager to
-allow easier periodic rotation of the ``Agent API Key`` with a constant
-``Project ID``)
+.. Note::
+    All the operations required to set up a Self-Signed CA can be automatically generated from
+    our :ref:`"How to configure a BigchainDB node" <how-to-configure-a-bigchaindb-node>` guide.
 
+5. Set Up a Self-Signed Certificate Authority
```

Comment:
This step doesn't say _how_ they should "Set Up a Self-Signed Certificate Authority" and it's my understanding that the generate_configs.sh bash script will do that. The instructions to run that are in the next page (linked), so I guess we could remove this step here?